### PR TITLE
fix(Argus reporting): fix ussue #6370

### PR DESCRIPTION
**Pull request by temichus**
now create-argus-test-run creates
a test stub with only Jenkins DATA
test DATA will be populated by SCTtest itself

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by temichus** [comment]
fix https://github.com/scylladb/scylla-cluster-tests/issues/6370

**Comment by temichus** [comment]
test: 
create argus run:
Screenshot from 2023-07-28 11-30-18
start test : 
Screenshot from 2023-08-01 10-58-56

File: sdcm/tester.py [hunk 1]
```diff
@@ -389,26 +389,14 @@ def _init_test_duration(self):
...
-            if not is_argus_run_exists:
-                self.test_config.argus_client().submit_sct_run(
-                    job_name=get_job_name(),
-                    job_url=get_job_url(),
-                    started_by=get_username(),
-                    commit_id=get_git_commit_id(),
-                    sct_config=self.params,
-                )
-                self.log.info("Initialized Argus TestRun with test id %s", self.test_config.argus_client().run_id)
+            self.test_config.argus_client().submit_sct_run(
```
Comment (soyacz) [inline sdcm/tester.py:392]: we no longer need to verify if argus run exists?

Comment (fruch) [reply sdcm/tester.py:392]: Sinse it needs to be called again to make sure we pass the SCT parameters.

Comment (k0machi) [reply sdcm/tester.py:392]: Affirmative, backend logic now will either create a non-existent run or update the existing one.

**Review by fruch — APPROVED** [review]
Much better solution

Side note: can you fix the unit test?

**Review by soyacz — APPROVED** [review]
LGTM

**Pull request merged**